### PR TITLE
Tests: Also use `datetime` column within `locations` test fixtures

### PR DESCRIPTION
**Pull request by amotl**
Hi,

this patch will be good to have before #437 in order to better outline specific changes to `date` vs. `datetime` return values.

All the values in the `datetime` column have been filled with the same value as in the corresponding `date` column. No other values have been changed.

With kind regards,
Andreas.

### CODE REVIEW

File: src/crate/client/doctests/client.txt [hunk 1]
```diff
@@ -213,7 +213,7 @@ supported, all other fields are 'None'::
     >>> pprint(result)
     ['Aldebaran',
      1373932800000,
-     None,
+     1373932800000,
```
Comment (matriv) [inline src/crate/client/doctests/client.txt:216]: maybe you can add also time info, down to millis?

Comment (amotl) [reply src/crate/client/doctests/client.txt]: Sure. a7fa1ee55f brings in a corresponding improvement.

Comment (matriv) [reply src/crate/client/doctests/client.txt]: As discussed, after that we need to add a column of `DATE` to test table `locations`, and we need also to add functionality to the client to truncate to start of day when using python date type, even if the table column (result) is a complete timestamp with time info.

**Review by matriv — APPROVED** [review]
LGTM, added a followup task/improvement as discussed.

**Review by mfussenegger — APPROVED** [review]

**Pull request merged**